sunbeam-python/sunbeam/plugins/observability/plugin.py

```diff
@@ -182,7 +182,7 @@ def run(self, status: Optional[Status] = None) -> Result:
         self.update_config(config)
         self.tfhelper.write_tfvars(tfvars)
 
-        self.update_status(status, "deploying application")
+        self.update_status(status, "deploying Grafana Agent")
```

Comment:
The repetition is not great:
```
Deploy Grafana Agent ... deploying Grafana Agent
```
This makes more sense to me:
```
Deploy Grafana Agent ... deploying application
```

```suggestion
        self.update_status(status, "deploying application")
```


Keep in mind that the whole line `update_status` is not necessary, it's mainly used when the step is doing multiple things and inform the user that, no, it's not stuck, something is going on.

You're free to follow my suggestion or removing the line altogether